src/client/visualizations/line-chart/charts/charts-per-series/extend.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as d3 from "d3";
+import { Dataset, Datum } from "plywood";
+import { Splits } from "../../../../../common/models/splits/splits";
+import { Unary } from "../../../../../common/utils/functional/functional";
+import { SPLIT } from "../../../../config/constants";
+
+export default function calculateExtend(dataset: Dataset, splits: Splits, getY: Unary<Datum, number>, getYP: Unary<Datum, number>): [number, number] {
```

Comment:
This function will change heavily when we start looking into split values instead of measures. Let's wait.